distribution/lib/Standard/Base/0.0.0-dev/src/Data/Array.enso

```diff
@@ -85,8 +85,8 @@ type Array
          elements, returning an an `Ordering` if the two elements are comparable
          or `Nothing` if they are not. If set to `Nothing` (the default argument),
          `Ordering.compare _ _` method will be used.
-       - on_incomparable: A `Problem_Behavior` specifying what should happen if
-         two incomparable values are encountered.
+       - on_problems: A `Problem_Behavior` specifying what should happen if two
```

Comment:
the parameter is still called `on_incomparable`. docs mismatch

Reply:
Yes is fixed now - was in progress a second ago.